### ISSUE TITLE
graphs do not display with shiny

### ISSUE THREAD

**Issue by fderyckel** [rstudio/dygraphs#54]
Hi all, 

Altough I have no problem using dygrpah from Rstudio normally, none of the dygraph display once I use shiny.  

I am using a MacbookPro 10.10 (but similar issue on mac desktop). 
I have tried to use both your example of shiny (the one here: https://rstudio.github.io/dygraphs/shiny.html), 
I have also try to use my own  (http://stackoverflow.com/questions/30176303/using-dygraph-with-shiny/30178925?noredirect=1#comment48475753_30178925)

But none of the graph display, although the whole shiny app display correctly.  (just the graph is not there :-( 

Version:
> version
               _                           
platform       x86_64-apple-darwin13.4.0   
arch           x86_64                      
os             darwin13.4.0                
system         x86_64, darwin13.4.0        
status                                     
major          3                           
minor          2.0                         
year           2015                        
month          04                          
day            16                          
svn rev        68180                       
language       R                           
version.string R version 3.2.0 (2015-04-16)
nickname       Full of Ingredients     

dygraph taken from github.  

Any idea why the graphs are not displaying? 
Thanks

**Comment by fderyckel** [comment]
And what is really weird, is that both this person (https://faidherbard.shinyapps.io/joburgdygraph/)  
and I (https://frankyd.shinyapps.io/shinyApps4dygraph/)
 have the same code, but mine is not displaying the graph!

**Comment by jjallaire** [comment]
Dygraphs definitely work in Shiny. I'd recommend making sure you are using
a recent version of Shiny and the other related packages like htmlwidgets.
You can try comparing the output of sessionInfo() for the two applications
to understand which packages may be of different versions.

**Comment by fderyckel** [comment]
Sorry I did not want to give the impression that dygraph did not work with shiny.  I know it works very well with it,that's why I wanted to use it ;-)  R is 3.2, dygraph is 0.4.4.1 and shiny is 0.11.1.  But then why when I publish my app on shinyapps.io, mine doesn't work.  Wouldn't the server of shinyapps.io have the same packages for all?

**Comment by jjallaire** [comment]
No, it uses whatever package versions were on your system at the time you
published the application.

**Comment by fderyckel** [comment]
Ok.  Thanks for that clarification.  But I still think I have the latest version of everything as I have updated all packages.  so I am still missing something ...

**Comment by jjallaire** [comment]
I just took that sample code and was able to run it locally as well as
successfully deploy and run it on shinyapps. Not sure what is going on for
you. Here is my sessionInfo:

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

other attached packages:
[1] dygraphs_0.4.3    shiny_0.11.1.9003

loaded via a namespace (and not attached):
 [1] digest_0.6.8      grid_3.1.3        htmltools_0.2.6
htmlwidgets_0.3.2 httpuv_1.3.2
 [6] jsonlite_0.9.16   lattice_0.20-30   magrittr_1.5      mime_0.2
 R6_2.0
[11] Rcpp_0.11.6       RJSONIO_1.2-0.2   rsconnect_0.3.78  rstudioapi_0.2
 tools_3.1.3
[16] xtable_1.7-4      xts_0.9-7         yaml_2.1.13       zoo_1.7-11

**Comment by skofieldO5** [comment]
Same problem here.

**Comment by mariafb** [comment]
Same problem here.
